core/server/models/api-key.js

```diff
@@ -67,6 +113,29 @@ const ApiKey = ghostBookshelf.Model.extend({
                 this.set('role_id', null);
             }
         }
+    },
+    onUpdated(model, attrs, options) {
```

Comment:
@kevinansfield Updated this from `onSaved` previously as that was adding `refreshed` events for new keys being added as well, which i think is undesirable. Using `OnUpdated` only adds `refreshed` for existing keys